SevenZip/SevenZipBase.cs

```diff
@@ -132,18 +121,6 @@ protected SevenZipBase(string password = "")
             _uniqueID = GetUniqueID();
         }
 
-        /// <summary>
-        /// Removes the UniqueID from the list.
-        /// </summary>
-        ~SevenZipBase()
-        {
-            // This lock probably isn't necessary but just in case...
-            lock (Identifiers)
-            {
-                Identifiers.Remove(_uniqueID);
-            }
-        }
-
```

Comment:
Great!

Reply:
I'm not 100% sure what the use of UniqueID is, while this implementation seems faster, it's also limited to ~4 million instances where the previous implementation was not

Can you shed some light on this?